js/app.js

```diff
@@ -87,12 +87,15 @@ function removeExistingImages() {
 
 function renderBusMallImages (item) {
   var busMallImagesParent = document.getElementById('busMallImagesParent');
+  var imgSpan = document.createElement('span');
```

Comment:
Yes, the span is a container for the image. I added the span to draw a uniform box around each image. In CSS I have set transitions on hover for both the image and the span.